Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -87,16 +87,34 @@ class RemoteConfigProperty<T> {
 
     var value: T {
         let remoteConfig = FIRRemoteConfig.remoteConfig()
+
+        // rcv is never nil even if there is no key.
+        // rcv.numberValue? never fails either (in light of an invalid number, it defaults to zero).
+        // So, we get the string value, and try parsing it ourselves.
         let rcv = remoteConfig[key]
-        if let value = convert(rcv) {
-            return value
+        if let string = rcv.stringValue {
+            if let value = convert(string) {
+                return value
+            } else if string != "" {
+                // i.e. no default exists, and no value is specified.
+                NSLog("RemoteConfigKeys: Existing value for \(key) is not a valid \(type(of: defaultValue))")
+            }
         }
-        if let rcv = remoteConfig.defaultValue(forKey: key, namespace: nil),
-            let value = convert(rcv) {
-            print("RemoteConfigKeys: Default value for \(key) from embedded plist")
+
+        // TODO: save the last successful value in UserDefaults.standard, then fall back to it here,
+        // instead of falling back to what's in the plist.
+
+        // Check to see if RemoteConfigDefaults.plist has a parseable default
```

Comment:
nit: this is slightly confusing. My understanding is that `remoteConfig[key]` will return a plist value if we call `setDefaultsFromPlistFile` before accessing the remote config.

Thus, its confusing because the comment "check to see if *.plist has a... default" because I'd think we'd already checked the default value in the original `remoteConfig[key]`.

To be clear, I understand what is happening and that this is a fallback: if `remoteConfig[key]` has been overwritten, we need to call `defaultValue` to get the plist value (though it's redundant in the case we've already checked the plist value).